tests/check_doctest_names.py

```diff
@@ -0,0 +1,201 @@
+"""A helper script to check names in doctests.
+
+The problem is that pytest doctests (following the stdlib doctest
+module) see the module-global namespace. So when a doctest looks
+like this:
+
+    Examples
+    --------
+    >>> import numpy
+    >>> import vtk
+    >>> import pyvista
+    >>> offset = np.array([0, 9])
+    >>> cells = np.array([8, 0, 1, 2, 3, 4, 5, 6, 7, 8, 8, 9, 10, 11, 12, 13, 14, 15])
+    >>> cell_type = np.array([vtk.VTK_HEXAHEDRON, vtk.VTK_HEXAHEDRON], np.int8)
+
+there will be a ``NameError`` when the code block is copied into Python
+because the ``np`` name is undefined. However, pytest and sphinx test
+runs will not catch this, as the np name is typically also available in
```

Comment:
```suggestion
runs will not catch this, as the ``np`` name is typically also available in
```
Ultra nitpick.

Reply:
Thanks. I added rst syntax as an afterthought and I missed this one.